shared/src/db/database.ts

```diff
@@ -83,6 +85,8 @@ class database extends Dexie {
             },
             { immediate: true },
         );
+
+        this.deleteExpired();
```

Comment:
As discussed, calling this in the constructor is fine, but I think it should be an interval, so that if someone has the website open for a while their expired documents get deleted as well.

Suggestion: Make `isCms` a property of the `database` class and set it in the constructor of `socket`: `db.isCms = ...`.

Then use that to check in your interval whether it's CMS or not.

Reply:
That does mean `isCms` should be `true` by default I think because otherwise in the first run it will delete documents if you're in the CMS - so perhaps this way is also not the best, but we can look into refactoring it if it works.